data/src/main/java/io/gnosis/kouban/data/utils/DateUtils.kt

```diff
@@ -0,0 +1,17 @@
+package io.gnosis.kouban.data.utils
+
+import java.util.*
+
+fun Date.onlyDate(): Date =
+    Calendar.getInstance().let { calendar ->
+        calendar.time = this
+        calendar.set(Calendar.HOUR_OF_DAY, 0)
+        calendar.set(Calendar.MINUTE, 0)
+        calendar.set(Calendar.SECOND, 0)
+        calendar.set(Calendar.MILLISECOND, 0)
+        calendar.time
+    }
+
+fun Date.afterOrEqual(other: Date?): Boolean = this == other || this.after(other)
```

Comment:
other should not be nullable. It will crash if you pass null as a value. The reason its not crashing now is the way its invoked in your if clause

Reply:
Thanks I totally missed that, I added some unit tests.